app-text/wordgrinder/wordgrinder-0.7.1.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A simple, Unicode-aware word processor that runs on the console"
+DESCRIPTION="WordGrinder is designed to get out of your way and let you write; \
+it does very little, but what it does it does it well. It supports basic paragraph \
+styles, basic character styles, basic screen markup, a menu interface that means you \
+don't have to remember complex key sequences, HTML import and export, and some other \
+useful features."
+HOMEPAGE="http://cowlark.com/wordgrinder/"
+COPYRIGHT="2007-2017 David Given"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/davidgiven/wordgrinder/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="54085af7963e1f67342bc0b1b20d1ccc75494f2e23d401b601cf3089acea747c"
+SOURCE_DIR="wordgrinder-$portVersion"
```

Comment:
haikuporter already sets a default value for SOURCE_DIR, and that default value is `wordgrinder-$portVersion`.
So this means you don't need to define SOURCE_DIR.
(Actually, we prefer to not have a SOURCE_DIR in the recipe when its default value is correct for the tarball.)

Reply:
Whoops. I got that fixed.
Also, for some reason, there are errors saying that devel:liblua5 and devel:libncurses6  doesn't exist. Does this mean I have to install these manually?

Reply:
Hmm, the packages are ncurses6 abd ncurses6_devel. I see that they provide lib:libncurses and devel:libncurses, so you just need to drop the 6.
BTW, to find what package provides devel:libncurses, you can use:
```
pkgman search devel:libncurses
```
It should tell that ncurses6_devel provides it. Then you can
```
pkgman install ncurses6_devel
```

Reply:
I've just checked for Lua:
devel:liblua is provided by lua_devel, so you can
```
pkgman install lua_devel
```
You will get Lua 5.3.